src/Microsoft.ML.Data/Scorers/MulticlassClassificationScorer.cs

```diff
@@ -450,7 +450,7 @@ internal static ISchemaBoundMapper WrapCore<T>(IHostEnvironment env, ISchemaBoun
                     trainSchema.Label.Value.GetKeyValues(ref value);
                 };
 
-            return LabelNameBindableMapper.CreateBound<T>(env, (ISchemaBoundRowMapper)mapper, type as VectorDataViewType, getter, AnnotationUtils.Kinds.SlotNames, CanWrap);
+            return LabelNameBindableMapper.CreateBound<T>(env, (ISchemaBoundRowMapper)mapper, type as VectorDataViewType, getter, AnnotationUtils.Kinds.TrainingLabelValues, CanWrap);
```

Comment:
>AnnotationUtils.Kinds.TrainingLabelValues [](start = 130, length = 41)

Hi @Ivanidzo4ka, could you explain this to me a bit more? It looks like we are no longer publishing these as slot names. Is that correct, or do I misread?

More broadly: I understand from the issue that there has been some regression from 0.11, but I am not certain I understand the nature of the regression completely (how did it regress, by which change?). That lack of knowledge makes this code difficult to review for me, even though the change itself is not too large.

Reply:
I ask you some time ago regarding nature of SlotNames, and what they are always should be Text instances. So I made change in `CanWrap` method which validates what KeyValues are actually TextInstances.
Which bring this bug.
I don't want to continue to use `SlotNames`, since they are limited by type. In same time `TrainingLabelValues` don't look like they have any restriction on data type of it content, so I just want to use it.